packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/update_pin_page.dart

```diff
@@ -0,0 +1,147 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/models.dart';
+import 'package:widget_toolkit/ui_components.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+import '../../app_extensions.dart';
+import '../../lib_router/blocs/router_bloc.dart';
+import '../../lib_router/router.dart';
+import '../bloc/pin_bloc.dart';
+import '../models/pin_code_arguments.dart';
+
+class UpdatePinPage extends StatefulWidget {
+  const UpdatePinPage({
+    this.pinCodeArguments = const PinCodeArguments(
+      title: '',
+      isSessionTimeout: false,
+    ),
+    super.key,
+  });
+
+  final PinCodeArguments pinCodeArguments;
+
+  @override
+  State<UpdatePinPage> createState() => _UpdatePinPageState();
+}
+
+class _UpdatePinPageState extends State<UpdatePinPage> {
+  @override
+  void initState() {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      context.read<PinBlocType>().events
+        ..setPinCodeType(widget.pinCodeArguments.isSessionTimeout)
+        ..temporaryDisableBiometrics(true);
+    });
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) => Builder(
+        builder: (context) => WillPopScope(
+          onWillPop: () async {
+            context.read<PinBlocType>().events.deleteSavedData();
+            return true;
+          },
+          child: Scaffold(
+            appBar: AppBar(
+              title: Text(
+                widget.pinCodeArguments.title.isEmpty
+                    ? context.l10n.libPinCode.updatePinPage
+                    : widget.pinCodeArguments.title,
+              ),
+            ),
+            extendBodyBehindAppBar: true,
+            body: SizedBox(
+              height: MediaQuery.of(context).size.height,
```

Comment:
Same here

Reply:
This was changed to use `MediaQuery.sizeOf(context).height`